databricks/koalas/tests/test_dataframe_spark_io.py

```diff
@@ -88,3 +88,23 @@ def test_parquet_write(self):
             # columns to appear first
             actual = ks.read_parquet(tmp)[self.test_column_order]
             self.assert_eq(actual.sort_values(by='f'), expected.sort_values(by='f'))
+
+    def test_spark_io(self):
+        with self.temp_dir() as tmp:
+            pdf = self.test_pdf
+            expected = ks.DataFrame(pdf)
+
+            # Write out partitioned by one column
+            expected.to_spark_io(tmp, format='json', mode='overwrite', partition_cols='i32')
+            # reset column order, as once the data is written out, Spark rearranges partition
```

Comment:
This line and 107 could start with a capital letter and end with a period, but that's rather cosmetic 😉